lib/stat_tracker.rb

```diff
@@ -24,7 +27,7 @@ def game_stats
   end
 
   def game_teams_stats
-    game_teams_data = CSV.read(@game_teams, {encoding: "UTF-8", headers: true, header_converters: :symbol, converters: :all})
+    game_teams_data = CSV.read(@game_teams, { encoding: 'UTF-8', headers: true, header_converters: :symbol, converters: :all })
```

Comment:
We should see if this works without the encoding: "UTF-8" argument.

Reply:
2nd that!